### ISSUE TITLE
Google Analytics not working?

### ISSUE THREAD

**Issue by jonfroehlich** [jonfroehlich/makeabilitylabwebsite#850]
![image](https://user-images.githubusercontent.com/1621749/70474256-20b71d00-1a87-11ea-9af7-1b60dff9c4d5.png)

Should be able to add the following to a base template:

```
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-79723676-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-79723676-1');
</script>
```

**Comment by jonfroehlich** [comment]
Working with @mechanicjay on this. Turns out, when we moved over to the new docker-compose setup to address #366, we didn't incorporate the config.ini file. This is now in docker-compose (as of this commit https://github.com/jonfroehlich/makeabilitylabwebsite/commit/f1c6b632803749d8d97a755b7c1328aa798b308e). I need to test to see if this resolves the problem and also, independently, whether we should move over to the new Google Analytics tracking code.
